OpenEdXMobile/res/layout/row_discussion_comments_response.xml

```diff
@@ -20,32 +20,20 @@
         android:orientation="vertical"
         android:padding="@dimen/discussion_responses_box_padding">
 
-        <LinearLayout
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:orientation="horizontal">
-
-            <include layout="@layout/row_discussion_user_profile" />
-
-            <Space
-                android:layout_width="0dp"
-                android:layout_height="0dp"
-                android:layout_weight="1" />
-
-            <TextView
-                android:id="@+id/discussion_comment_count_report_text_view"
-                style="@style/discussion_responses_small_text"
-                android:layout_gravity="end"
-                android:background="?android:selectableItemBackground"
-                android:clickable="true"
-                android:padding="@dimen/discussion_responses_box_padding"
-                tools:text="8 comments" />
-
-        </LinearLayout>
+        <include layout="@layout/row_discussion_user_profile" />
 
         <TextView
             android:id="@+id/discussion_comment_body"
             style="@style/discussion_regular_text"
             tools:text="This is the text of a response." />
+
+        <TextView
+            android:id="@+id/discussion_comment_count_report_text_view"
+            style="@style/discussion_responses_small_text"
+            android:layout_gravity="end|right"
+            android:background="?android:selectableItemBackground"
+            android:clickable="true"
+            android:padding="@dimen/discussion_responses_box_padding"
+            tools:text="8 comments" />
     </LinearLayout>
-</android.support.v7.widget.CardView>
+</android.support.v7.widget.CardView>
```

Comment:
Github likes newline at end of files. Also, can you auto-format this class too?

Reply:
@miankhalid sure